### PR TITLE
Khayaal 7 divider and heading

### PR DESCRIPTION
**Pull request by preethisun0801**
our teams divider and heading

### CODE REVIEW

File: src/Components/OurTeams/DividerAndHeading.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+.divhead {
+  font-family: "Roboto", sans-serif;
+  font-weight: 700;
+  line-height: 38px;
+}
+.divider {
+  left: 0%;
```
Comment (roshan1999) [inline src/Components/OurTeams/DividerAndHeading.css:8]: dont use absolute, use flexbox

**Review by roshan1999 — CHANGES_REQUESTED** [review]
Address all changes from leader description component over here as well

**Comment by preethisun0801** [comment]
flexbox used for divider, font size and color changed for generalisation

**Comment by preethisun0801** [comment]
[KHAYAAL-7.docx](https://github.com/roshan1999/interns-khayaal-app/files/8771475/KHAYAAL-7.docx)

**Review by roshan1999 — APPROVED** [review]

**Pull request merged**